app-config.yaml

```diff
@@ -20,5 +20,9 @@ proxy:
 organization:
   name: Spotify
 
+github-actions:
+  owner: CircleCITest3
+  repo: try-ssr
```

Comment:
Don't see the point of having this here tbh? 😅 , might as well just hardcode it in the plugin as it's gonna have to vary for every entity anyway.

Reply:
Going to just recreate this branch from scratch :) Too much leftovers from underlying branch